tests/dragonfly/replication_test.py

```diff
@@ -2626,3 +2626,34 @@ async def test_replica_of_replica(df_factory):
         await c_replica2.execute_command(f"REPLICAOF localhost {replica.port}")
 
     assert await c_replica2.execute_command(f"REPLICAOF localhost {master.port}") == "OK"
+
+
+@pytest.mark.asyncio
+@pytest.mark.slow
+async def test_big_containers(df_factory):
+    master = df_factory.create(proactor_threads=4)
+    replica = df_factory.create(proactor_threads=4)
+
+    df_factory.start_all([master, replica])
+    c_master = master.client()
+    c_replica = replica.client()
+
+    logging.debug("Fill master with test data")
+    seeder = StaticSeeder(
+        key_target=20,
+        data_size=4000000,
+        collection_size=1000,
```

Comment:
1. one test case for big collection with more than kMaxBlobLen = 4092 elements , to check the flow in rdb load for big containers, the size of elemets should be small in this test , testing #3760 
2. second test case for big values of size ~10K, we can have 2 or 3 items in the collection in this test , testing https://github.com/dragonflydb/dragonfly/pull/3914